spec/memoize_query_spec.rb

```diff
@@ -446,9 +446,13 @@ def expect_to_eq_with_or_without_redis
   context 'when the queries have NOT' do 
     let!(:record) { Site.create!(a: 1, b: 1) }
     let!(:relation1_with_only_not) { Site.where.not(a: 2) }
-    let!(:relation2_with_only_not) { Site.where.not(a: 2, b: 1) }
+    let!(:relation2_with_only_not) { Site.where.not(a: 2, b: 2) }
     let!(:relation3_with_only_not) { Site.where.not(a: 2).where.not(b: 1) }
     let!(:relation_with_not_and_other) { Site.where.not(a: 2).where(b: 1) }
+    # where.not(a: ..., b:...) relation is treated differently in Rails 6.0, Rails 6.1 and before Rails 6
```

Comment:
This comment is slightly confusing. The ones after L471 seem to be clearer to me.

Reply:
changed a bit, let me know if that is clear enough